AppController/djinn.rb

```diff
@@ -5606,7 +5542,7 @@ def try_to_scale_up(app_name, delta_appservers)
 
         # Now we do a similar calculation but for the current amount of
         # available memory on this node. First convert bytes to MB
-        host_available_mem = node['memory']['available']/1024/1024
+        host_available_mem = Float(node['memory']['available']/1024/1024)
```

Comment:
I noticed that 1024/1024 is used few times: perhaps we should use a contant MB? Or something similar?